config/brakeman.ignore

```diff
@@ -1,36 +1,48 @@
 {
   "ignored_warnings": [
     {
-      "warning_type": "Cross-Site Scripting",
```

Comment:
As I had to add the modified SQL queries to the Brakeman ignore file, I also did an automated cleanup of legacy ignored warnings that are no longer needed. Hence all the code removal.

This is automated by `brakeman -I` tool.